src/components/Dropdown/Dropdown.story.jsx

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { generateVariantsFor } from '../../utils/propsVariants';
+import Dropdown from './Dropdown';
+import iconNames from '../Icon/Icons';
+
+const stories = storiesOf('Dropdown', module);
+
+stories.add(
+  'Dropdown',
+  () => {
+    const variants = generateVariantsFor({
+      topText: [null, '', 'Resources'],
+      topIcon: [null, 'edit'],
+      items: [
+        [],
+        [
+          { value: 'Data Resource Protal', icon: 'file-size', onClick: () => {} },
+          { value: 'Website', icon: 'website', onClick: () => {} },
+          { value: 'Studies and Access', icon: 'study', onClick: () => {} },
+          { value: 'Support', icon: 'info', onClick: () => {} },
+          { value: 'Contact', icon: 'email', onClick: () => {} },
+        ],
+      ],
+      open: [true, false, null],
+    });
+    const items = [
+      { value: 'Data Resource Protal', icon: 'file-size', onClick: () => {} },
+      { value: 'Website', icon: 'website', onClick: () => {} },
+      { value: 'Studies and Access', icon: 'study', onClick: () => {} },
+      { value: 'Support', icon: 'info', onClick: () => {} },
+      { value: 'Contact', icon: 'email', onClick: () => {} },
```

Comment:
add the new props to the items and change `onClick` to 
```js 
onClick: e => {e.preventDefault()}
```
which will keep storybook from reloading when someone clicks a link